core/node/consistency_checker/src/lib.rs

```diff
@@ -312,13 +312,21 @@ pub struct ConsistencyChecker {
     max_batches_to_recheck: u32,
     sleep_interval: Duration,
     l1_client: Box<DynClient<L1>>,
+    migration_setup: Option<MigrationSetup>,
```

Comment:
nit: let's include "gateway" word in struct or var name so it's clear it's migration to gateway

Reply:
It's not necessarily to gateway. Even though for now we don't support it, we can be migrating to other settlement layers (Gateway->L1, or sideways)